
Resync sparc grub-emu using powerpc version.  This will disappear when
grub-emu declarations are unified.

Index: conf/sparc64-ieee1275.rmk
===================================================================
--- conf/sparc64-ieee1275.rmk	(revision 2375)
+++ conf/sparc64-ieee1275.rmk	(working copy)
@@ -97,42 +97,44 @@
 
 # For grub-emu
 util/grub-emu.c_DEPENDENCIES = grub_emu_init.h
-grub_emu_SOURCES = commands/boot.c commands/cat.c commands/cmp.c 	\
+grub_emu_SOURCES = commands/minicmd.c commands/cat.c commands/cmp.c 	\
 	commands/configfile.c commands/help.c				\
 	commands/search.c commands/handler.c commands/test.c 		\
 	commands/ls.c commands/blocklist.c commands/hexdump.c		\
-	commands/probe.c commands/xnu_uuid.c				\
-	lib/hexdump.c commands/halt.c commands/reboot.c			\
+	lib/hexdump.c commands/halt.c commands/reboot.c 		\
+	commands/gptsync.c commands/probe.c commands/xnu_uuid.c		\
 	disk/loopback.c							\
 	\
-	fs/affs.c fs/cpio.c fs/ext2.c fs/fat.c fs/hfs.c			\
+	fs/affs.c fs/cpio.c fs/fat.c fs/ext2.c fs/hfs.c			\
 	fs/hfsplus.c fs/iso9660.c fs/udf.c fs/jfs.c fs/minix.c		\
 	fs/ntfs.c fs/ntfscomp.c fs/reiserfs.c fs/sfs.c			\
 	fs/ufs.c fs/xfs.c fs/afs.c fs/tar.c				\
 	\
 	io/gzio.c							\
 	kern/device.c kern/disk.c kern/dl.c kern/elf.c kern/env.c	\
-	kern/err.c kern/file.c kern/fs.c kern/loader.c kern/main.c	\
+	kern/err.c kern/file.c kern/fs.c commands/boot.c kern/main.c	\
 	kern/misc.c kern/parser.c kern/partition.c kern/reader.c	\
 	kern/rescue_reader.c kern/rescue_parser.c			\
-	kern/term.c fs/fshelp.c						\
-	kern/list.c kern/handler.c					\
-	lib/arg.c normal/cmdline.c 					\
-	normal/completion.c		 				\
-	normal/main.c normal/menu.c 					\
+	kern/term.c kern/list.c kern/handler.c fs/fshelp.c		\
+	kern/command.c kern/corecmd.c commands/extcmd.c			\
+	lib/arg.c normal/cmdline.c normal/datetime.c 			\
+	normal/completion.c normal/misc.c		 		\
+	normal/handler.c normal/autofs.c normal/main.c			\
+	normal/menu.c 							\
 	normal/menu_text.c						\
-	normal/menu_entry.c normal/menu_viewer.c normal/misc.c 		\
+	normal/menu_entry.c normal/menu_viewer.c 	 		\
 	normal/color.c							\
-	script/sh/main.c script/sh/execute.c script/sh/function.c       \
-	script/sh/lexer.c script/sh/script.c 			        \
+	script/sh/main.c script/sh/execute.c script/sh/function.c	\
+	script/sh/lexer.c script/sh/script.c				\
 	partmap/amiga.c	partmap/apple.c partmap/pc.c partmap/sun.c	\
 	partmap/acorn.c							\
 	util/console.c util/hostfs.c util/grub-emu.c util/misc.c	\
 	util/hostdisk.c util/getroot.c					\
-	util/sparc64/ieee1275/misc.c					\
+	util/sparc64/ieee1275/misc.c					\
 	\
 	disk/raid.c disk/raid5_recover.c disk/raid6_recover.c		\
 	disk/mdraid_linux.c disk/dmraid_nvidia.c disk/lvm.c		\
+	commands/parttool.c parttool/pcpart.c				\
 	grub_script.tab.c grub_emu_init.c
 
 grub_emu_LDFLAGS = $(LIBCURSES)
